vault/request_handling.go

```diff
@@ -1415,6 +1415,7 @@ func (c *Core) handleLoginRequest(ctx context.Context, req *logical.Request) (re
 			return nil, nil, err
 		}
 		if isloginUserLocked {
+			c.startLockoutLogger()
```

Comment:
🤔 It seems like the lockout logger would benefit from being a singleton routine. Right now, if multiple users are locked out it appears that we would have multiple logout loggers running simultaneously.

Reply:
startLockoutLogger checks to see if we have a cancel func implying we already have a goroutine running, and will fast fail out without spinning up a new goroutine. https://github.com/hashicorp/vault/blob/5b635ab88b6110ebbc7eca1c4fab8bec1a03f4d1/vault/core.go#L3614-L3617

Reply:
Ah, I missed that, whoops! It also looks like you nil out the cancel upon having no locked out users. Thanks for the clarification